l2tdevtools/review_helpers/github.py

```diff
@@ -92,6 +92,34 @@ def CreatePullRequestReview(
 
     return True
 
+  def AssignPullRequest(
+      self, pull_request_number, access_token, assignees):
+    """Assigns a GitHub pull request.
+
+    Args:
+      pull_request_number (int): GitHub issue number of the pull request.
+      access_token (str): github access token.
+      assignees (list[str]): github usernames to assign.
+
+    Returns:
+      bool: True if the review was created.
```

Comment:
=> "True if the assignees were successfully added."

Reply:
Done